velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -248,6 +249,41 @@ TEST_F(StringImplTest, charToCodePoint) {
   testExpectDeath("");
 }
 
+TEST_F(StringImplTest, stringToCodePoint) {
+  auto testValidInput = [](const std::string& charString,
+                           const std::vector<int32_t> expectedCodePoint,
+                           const bool isAscii = true) {
+    std::vector<int32_t> res;
+    if (isAscii) {
+      res = stringToCodePoint</*isAscii*/ true>(StringView(charString));
+    } else {
+      res = stringToCodePoint</*isAscii*/ false>(StringView(charString));
+    }
+    for (int i = 0; i < res.size(); i++) {
+      ASSERT_EQ(res.at(i), expectedCodePoint.at(i));
+    }
+  };
+  testValidInput("", {});
+  testValidInput("h", {0x0068});
+  testValidInput("hello", {0x0068, 0x0065, 0x006C, 0x006C, 0x006F});
+  testValidInput("hïllo", {0x0068, 0x00EF, 0x006C, 0x006C, 0x006F}, false);
+  testValidInput("ü", {0x00FC}, false);
+  testValidInput("\u840C", {0x840C}, false);
+
+  VELOX_ASSERT_THROW(
+      stringToCodePoint</*isAscii*/ false>(StringView("\xA9")),
```

Comment:
Would you add some tests where invalid characters appear in the middle of the string?